### PR TITLE
information about triple data as JSON format is added to NEEM acquisition

### PR DESCRIPTION
**Pull request by AbhijitVyas**

### CODE REVIEW

File: docs/NEEM-handbook/content/acquisition/acquisition.tex [hunk 1]
```diff
@@ -53,6 +53,21 @@ \section{Data Structure}
 	\item[\textbf{triples}] The triples collection contains all triples which were asserted into the knowledge base during run time.
 \end{description}
 
+\subsection{Triple data as JSON object}
+	One can provide triple data into JSON format logged in their experiment, where triples are defined in standard subject, predicate and object format. Subject and objects can be defined with valid IRI, which is pointing to concepts or individuals defined by ontology. An individual name is the combination of concept name and random string connected with an underscore. An object can be also other data types, such as a number, string, boolean, etc. Whereas, the predicate is defined by IRI pointing to property concepts mentioned in  ontology. One can also provide additional time scope fields `since` and `until` to indicate that the given triple is valid for the given time scope. An example of such JSON object is given below:
```
Comment (daniel86) [inline docs/NEEM-handbook/content/acquisition/acquisition.tex:57]: Better: "Triple data can also be provided in form of ..."
"One can .." seems not formal enough to me.

Comment (daniel86) [inline docs/NEEM-handbook/content/acquisition/acquisition.tex:57]: "logged in *their* experiment" is invalid grammar.
I would suggest to remove "logged in their experiment"

Comment (daniel86) [inline docs/NEEM-handbook/content/acquisition/acquisition.tex:57]: "where triples are defined in standard subject, predicate and object format"
what is the "standard format", I can not get the meaning of this sentence

Comment (daniel86) [inline docs/NEEM-handbook/content/acquisition/acquisition.tex:57]: "Subject and objects" singular/plural mix-up
either both singular or both plural

Comment (daniel86) [inline docs/NEEM-handbook/content/acquisition/acquisition.tex:57]: "can be defined with valid IRI"
what is a "valid" IRI, what does "defines" mean here?

Comment (daniel86) [inline docs/NEEM-handbook/content/acquisition/acquisition.tex:57]: "defined by ontology" bad grammar. *an* is missing, or say "the SOMA ontology"
I would also write "defined in"

Comment (daniel86) [inline docs/NEEM-handbook/content/acquisition/acquisition.tex:57]: "combination of **a** concept name and **a** random string"

Comment (daniel86) [inline docs/NEEM-handbook/content/acquisition/acquisition.tex:57]: "An object can be also other data types" sounds odd.
If it is a number, then it is not an object!
So you rather say that triples can not only link subject with object, but also can link subject with data value which is represented using one of the base types string, bool, number

Comment (daniel86) [inline docs/NEEM-handbook/content/acquisition/acquisition.tex:57]: "the predicate is defined by IRI"
first, again **an** is missing
second, the predicate is not **defined** by the IRI, it is **named** by the IRI.
Defining is something different

Comment (daniel86) [inline docs/NEEM-handbook/content/acquisition/acquisition.tex:57]: "An example of such **an** JSON object"

File: docs/NEEM-handbook/content/acquisition/acquisition.tex [hunk 2]
```diff
@@ -53,6 +53,21 @@ \section{Data Structure}
...
+   "s": "http://www.ease-crc.org/ont/SOMA.owl#Salad_PMRVYPJH",
+   "p": "http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#hasRole",
+   "o": "http://www.ease-crc.org/ont/SOMA.owl#Patient_YGJUVNDR",
+   "since": 27.739,
+   "until": 29.075
+ }
+]
+\end{lstlisting}
+
+It is important to note that, this is an intermediate data format which is not stored into database but is useful for providing data into agreed common format which can be accessed and inferred into knowledge-base.
```
Comment (daniel86) [inline docs/NEEM-handbook/content/acquisition/acquisition.tex:70]: "which is not stored into database but is useful for providing data into agreed common format which can be accessed and inferred into knowledge-base." this seems odd
-->
"which is not equivalent with how the NEEM narrative is actually stored in databases.
The format described here rather serves as an easy-to-use interchange format."

**Review by daniel86 — CHANGES_REQUESTED** [review]

File: docs/NEEM-handbook/content/acquisition/acquisition.tex (hunk 1 above)
Comment (AbhijitVyas) [reply docs/NEEM-handbook/content/acquisition/acquisition.tex:57]: S, P, O is triple standard format.

Comment (AbhijitVyas) [reply docs/NEEM-handbook/content/acquisition/acquisition.tex:57]: valid IRI which is pointing to concepts or individuals defined in ontology

Comment (AbhijitVyas) [reply docs/NEEM-handbook/content/acquisition/acquisition.tex:57]: I can remove word "valid" in order to avoid further confusion.

**Review by daniel86 — APPROVED** [review]

**Pull request merged**